### ISSUE TITLE
Documentation request: 

### ISSUE THREAD

**Issue by sesam** [GoogleCloudPlatform/functions-framework-nodejs#226]
The README mentions setting signature several times, but clicking around gave no info how to set signature. Searching a bit, I found a solution for Python functions: https://github.com/GoogleCloudPlatform/functions-framework-python/pull/78/commits/cd2f753f7468da56c1702780f8f0a7fa3ef2ac6b
I'd like to see a documentation page about this signature concept, and how to do it for all supported languages and signature types. Maybe mention deprecated signatures, if "event" is one such?
Maybe this info is somewhere in the docs folder, then just linking from the README would be perfect.

**Comment by grant** [comment]
Hey @sesam,

We have a section on how to set the signature and configure the FF:

https://github.com/GoogleCloudPlatform/functions-framework-nodejs#configure-the-functions-framework

There's an example using flags too in the README.

---

I'm not sure how that is skipped or what we should add.

**Comment by sesam** [comment]
tl;dr Sorry for the noise.

Hm, I'm not sure now how I failed to understand the concept of signature. I probably arrived reading about the default (http) and somehow ended up googling. Then I found some info in the other repo, but not much more.
Probably I just failed at reading, or maybe I was on some other page that's not part of this git repo.
Sorry for the noise.
